pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -39,33 +39,39 @@ var serviceProtocolMap = map[uint8]corev1.Protocol{
 }
 
 type ConntrackConnectionStore struct {
-	flowRecords          *flowrecords.FlowRecords
 	connDumper           ConnTrackDumper
 	v4Enabled            bool
 	v6Enabled            bool
 	networkPolicyQuerier querier.AgentNetworkPolicyInfoQuerier
 	pollInterval         time.Duration
-	connectionStore
+	ConnectionStore
+	expirePriorityQueue *flowexporter.ExpirePriorityQueue
+	activeFlowTimeout   time.Duration
+	idleFlowTimeout     time.Duration
```

Comment:
These seem to be common with `DenyConnectionStore` and can be added directly into the connection store.